app/src/main/java/com/example/coinage/fragments/GoogleSheetsFragment.java

```diff
@@ -35,7 +35,6 @@
 import com.google.api.client.json.gson.GsonFactory;
```

Comment:
Re: the git stuff we were talking about earlier, I see four commits in the PR - this is no problem (maybe even ideal) as long as you "Squash and Merge" so that they don't go into master as 4 separate commits. It's in the dropdown on the "merge pull request" button